recipe/patch_yaml/scvi-tools.yaml

```
@@ -9,3 +9,16 @@ then:
   - tighten_depends:
       name: lightning
       upper_bound: "2.2"
+---
+# scvi-tools 1.1.6.post2 requires anndata<=0.10.8
+# https://github.com/scverse/scvi-tools/pull/2971
+# https://github.com/conda-forge/scvi-tools-feedstock/pull/57#issuecomment-2353371071
+if:
+  name: scvi-tools
+  version: "1.1.6.post2"
+  build_number: 0
+  timestamp_lt: 1726588981000
+then:
+  - tighten_depends:
+      name: anndata
+      upper_bound: "0.10.9"
```

For future reference, this would also exclude pre-releases
```suggestion
      upper_bound: "0.10.9a0"
```